frontend/src/components/mobDropdownMenu/mobDropdownMenu.css

```diff
@@ -19,12 +19,22 @@
     top: 50% !important;
     left: 50% !important;
     transform: translate(-50%, -50%) !important;
-    background: rgba(0, 0, 13, 0.64);
-    z-index: 9999;
+    background: rgba(0, 0, 13, 1);
+    z-index: 50 !important;
     pointer-events: auto;
     top: 280px !important;
   }
 
+  .send-to-back {
+    z-index: -5;
+    position: relative;
+  }
+
+  .send-to-ultimate-back {
+    z-index: -10;
+    position: relative;
+  }
+
```

Comment:
this solution requires to assign those classes on each new page and to multipli elements, which is not ideal
Please create a solution that does not involve changes in other pages/component apart from dropdown one.

Solution should be applied to both (2) dropdowns

Reply:
Alright, i'm on it.